src/files.js

```diff
@@ -242,11 +242,29 @@ FOAM_FILES([
   { name: "foam/blob/BlobJava", flags: ['java'] },
   { name: "lib/node/json_dao", flags: ['node'] },
   { name: "lib/utf8" },
+  { name: "foam/nanos/auth/EnabledAware" },
```

Comment:
Most of the nanos files are already listed in src/foam/nanos/nanos.js Can we add require('../../src/foam/nanos/nanos.js') to test/helpers/assert.js similar to how we load foam.js there?